### PR TITLE
Feat/notification bulk

### PR DESCRIPTION
**Pull request by Davidmattei**
|Q              |A  |
|---------------|---|
|Bug fix?       |n|
|New feature?   |y|	
|BC breaks?     |n|	
|Deprecations?  |n|	
|Fixed tickets  |n|

### CODE REVIEW

File: src/Command/Notification/BulkActionCommand.php [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+        return 0;
+    }
+
+    /**
+     * @param array<mixed> $query
+     *
+     * @return DocumentInterface[]
+     */
+    private function searchDocuments(Environment $environment, array $query): array
```
Comment (theus77) [inline src/Command/Notification/BulkActionCommand.php:126]: Why nott returning an iterator? You load all document in memory

File: src/Entity/ContentType.php [hunk 2]
```diff
@@ -1149,6 +1149,15 @@ public function getEnvironment()
...
 
+    public function giveEnvironment(): Environment
+    {
+        if (null === $this->environment) {
+            throw new \RuntimeException('Environment not found!');
+        }
+
+        return $this->environment;
+    }
+
```
Comment (theus77) [inline src/Entity/ContentType.php:1160]: I love it

File: src/Entity/Template.php [hunk 3]
```diff
@@ -708,12 +708,7 @@ public function setContentType(ContentType $contentType = null)
         return $this;
     }
 
-    /**
-     * Get contentType.
-     *
-     * @return \EMS\CoreBundle\Entity\ContentType
-     */
-    public function getContentType()
+    public function getContentType(): ContentType
```
Comment (theus77) [inline src/Entity/Template.php:711]: Did you try to create a brand new template? I'm afraid that the field migth be null in that case

Comment (Davidmattei) [reply src/Entity/Template.php]: Fixed add a giveContentType and reverted this change

File: src/Command/Notification/BulkActionCommand.php (hunk 1 above)
Comment (Davidmattei) [reply src/Command/Notification/BulkActionCommand.php]: Fixed used the count on the elasticSearch service

File: src/Entity/ContentType.php (hunk 2 above)
Comment (Davidmattei) [reply src/Entity/ContentType.php:1160]: Me too

**Review by theus77 — APPROVED** [review]

**Pull request merged**